plugins/repository-azure/src/main/java/org/opensearch/repositories/azure/AzureStorageSettings.java

```diff
@@ -325,7 +327,7 @@ public String toString() {
         sb.append(", timeout=").append(timeout);
         sb.append(", endpointSuffix='").append(endpointSuffix).append('\'');
         sb.append(", maxRetries=").append(maxRetries);
-        sb.append(", proxy=").append(proxy);
+        sb.append(", proxySettings=").append(proxySettings != ProxySettings.NO_PROXY_SETTINGS ? "PROXY_SET" : "PROXY_NOT_SET");
```

Comment:
This is a nitpick, but things like `PROXY_SET` imply some kind of variable. Since this is a log, maybe just use "set" and "not set"?